security/crowdsec/files/pkg-deinstall.in

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+case $2 in
+       DEINSTALL)
+               %%PREFIX%%/etc/rc.d/crowdsec stop
```

Comment:
We could either use the command `service` or the rc script here both are valid options.

Reply:
"service crowdsec stop", ok. I was doing like other projects in /src/ports